betelgeuse.py

```diff
@@ -20,11 +20,17 @@
 import ssl
 import testimony
 import time
+import sys
 
 from collections import Counter
-from pylarion.exceptions import PylarionLibException
-from pylarion.work_item import TestCase, Requirement
-from pylarion.test_run import TestRun
+try:
+    from pylarion.exceptions import PylarionLibException
+    from pylarion.work_item import TestCase, Requirement
+    from pylarion.test_run import TestRun
+except ImportError as exception:
```

Comment:
What is the real benefit of this? If the module is not installed it will raise `ImportError: No module named pylarion.<submodule>` which states that the pylarion is not available and will already exit the process.

The main point here is that doing like so will mask other import errors, for example you have pylarion installed and some of its dependencies are missing then it will raise an `ImportError` and will be stated that pylarion is not available which is not correct.

This can be confirmed by doing a simple example, create a file `mymodule.py`:

```python
import missingmodule
```

And then test on the python interpreter:

```
>>> try:
...     import mymodule
... except ImportError:
...     print('mymodule is not available')
... 
mymodule is not available
```

That will be a wrong information since the real cause is:

```
>>> import mymodule
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "mymodule.py", line 1, in <module>
    import someother
ImportError: No module named someother
```